src/mame/psikyo/psikyo.cpp

```diff
@@ -29,14 +29,9 @@ Tengai              (J) 1996    SH404          SH404 has MCU, ymf278-b for sound
 
 To Do:
 
-- All games uses PORT_VBLANK and legacy screen parameters (which is already
-  bad per-se), with also naive and unlikely measurements (i.e. exactly 59.30).
-  The most blunt examples of something being wrong with timings are with
-  Gunbird and Tengai: they both have FOUR frames of input lag, the real thing
-  doesn't sport anything like that.
-  Given the above, all games are marked with MACHINE_IMPERFECT_TIMING until
-  somebody provides accurate H/Vsync signals for at least one game of this
-  driver.
+- Some games use naive/unlikely measurements (exactly 59.30), which should be verified.
+  Tengai and Strikers 1945 has accurate measurements from PCB. Maybe these could be used
+  for other titles too?
```

Comment:
Shoule be “…have accurate measurements…”

Reply:
Done